app/src/main/java/org/xbmc/kore/ui/sections/remote/RemoteActivity.java

```diff
@@ -362,9 +365,13 @@ private static ExecutorService getShareExecutor() {
      * Handles the intent that started this activity, namely to start playing something on Kodi
      * @param intent Start intent for the activity
      */
-    private void handleStartIntent(Intent intent) {
+    protected void handleStartIntent(Intent intent) {
+        handleStartIntent(intent, false);
```

Comment:
Shouldn't `queue` be set to `true` by default? The previous behaviour queued the files. I'll make a brief comment on the code pointing this out.

Reply:
Ignore this and the other comment. Noticed now that it has nothing to do with what i wrote, and is only used to display a message.

Reply:
> is only used to display a message.

Well... it calls `Player.Open` instead of `Playlist.Add`.

I have changed the behaviour of `Play with Kodi` because the word _play_ sounds like playing immediately, and queuing is a bit weird. When I was using Kore for the first time I thought it wasn't working, not realizing it was queuing the medias.